tiup: add 3 docs cluster-rename/cluster-restart/dm-import (#5323)
pingcap/docs: tiup/tiup-component-cluster-rename.md

---
title: tiup cluster rename
---

# tiup cluster rename

The cluster name is specified when [the cluster is deployed](/tiup/tiup-component-cluster-deploy.md). If you want to change the cluster name after the cluster is deployed, you can use the command `tiup cluster rename`.

> **Note:**
>

> > If the tiup cluster is configured with the `dashboard_dir` field of `grafana_servers`, when you execute the command `tiup cluster rename` to rename the cluster, the following additional steps are required:

Reviewer: ```suggestion
> If the `dashboard_dir` field of `grafana_servers` is configured for the TiUP cluster, after you execute the command `tiup cluster rename` to rename the cluster, the following additional steps are required:
```
